.github/workflows/build.yml

```diff
@@ -66,7 +66,7 @@ jobs:
       - shell: bash
         run: pip3 install virtualenv
 
-      - uses: actions/setup-haskell@v1
+      - uses: actions/setup-haskell@v1.1
```

Comment:
Based off of what @jared-w has told me, `actions/setup-haskell` is out of date and `haskell/actions/setup` should be used instead. I'm not sure about the precise version number, but I believe `@v1` should suffice.

Reply:
I just noticed his PR that does essentially the same thing as this one. :) I guess we should only merge one of them!

Reply:
Updated